index.html

```diff
@@ -30,6 +30,44 @@ <h2>What I do</h2>
       <p>
         <a href="https://www.linkedin.com/in/olysu/">view full cv</a>
       </p>
+      <!--work section-->
+      <h2>work I've done</h2>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 work-1">
+            <div class="img">
+              <img
+                src="./imgs/buildkit.jpg"
+                alt="logo of the tech company Buildkit"
+              />
+            </div>
+            <h3>Improve how we navigate</h3>
+          </div>
+          <div class="col-6 work-2">
+            <div class="img">
+              <img src="./imgs/worksafe.jpg" alt="logo of worksafe" />
+            </div>
+            <h3>Improve how we navigate</h3>
```

Comment:
I don't think we should use `h3` because, although it's styled like a heading, without paragraph text then its use is not a heading. 

I'm opinionated that styling and markup should be decoupled.